docs/tutorials/build-an-outdoor-rover.md

```diff
@@ -0,0 +1,304 @@
+Finally, one of the two motors we are connecting will need the "Direction Flip" attribute set to "true".
+It's ok if you are not sure which one, you can always switch them later.
+Regardless, once you've configured the "left" motor, repeat for the "right" - just make sure you're mapping the correct GPIO pins for each.
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/motor-setup.png" alt="Motor setup." title="Motor setup." />
+</div>
+
+At this point, you can try controlling your motors with the Viam App.
+Navigate to the control tab.
+If all is set up correctly, you'll see cards for your left and right motors on this page.
+Open one up, and if your rover is in a safe position (on the ground or propped up), try running one!
+
+Caution: Be aware that these are powerful motors, so you don't want to start at 100% power right away.
+
+{{% alert title="Caution" color="caution" %}}
+Caution: Be aware that these are powerful motors, so you don't want to start at 100% power right away.
+{{% /alert %}}
+
+If the motors don't appear on the control page, or are not running when expected, first check the "LOGS" tab to see if you see any errors - which would likely point out a bad configuration (a typo, a misplaced or missing attribute, etc).
+If not, double check your wiring and how it maps to the GPIO pins you've configured.
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/motor-control.png" alt="Motor control." title="Motor control." />
+</div>
+
+#### Base component
+
+Now, let's configure a "base" component that references both of our motors.
+This will allow us to control our robot's movement through a single interface.
+Add a new component of type "base", model "wheeled" - which represents a wheeled base with two powered wheels.
+The "left" and "right" attributes reference the motors you've configured by name.
+You can measure your wheel's circumference and width between wheels, which will allow the Viam platform to properly calculate how to steer your base.
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/base-setup.png" alt="Base setup." title="Base setup." />
+</div>
+
+## Take your Boxbot for a drive
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/completed-rover.jpg"  style="float:right;margin-left:1em;" alt="Completed rover with mounted solar panel." title="Completed rover with mounted solar panel." width="350" />
+</div>
+
+You can now have some real fun.
+Try driving your rover by keyboard.
+Drive your [robot progammatically](/tutorials/viam-rover/try-viam-sdk/), use a [color detector](/tutorials/viam-rover/try-viam-color-detection/) or a [machine learning detector](/services/vision/#detection) to interact with the environment.
```

Comment:
```suggestion
Drive your [robot progammatically](/tutorials/viam-rover/try-viam-sdk/), use a [color detector](/tutorials/viam-rover/try-viam-color-detection/) or a [object detector](/services/vision/#detection) to interact with the environment.
```

Reply:
machine learning is less specific and more of a searched-for term (but honest here)

Reply:
machine learning detector just made it sound like you are detecting ml models. maybe "machine learning object detector"?

Reply:
also, i just realized you have a typo on programmatically